### ISSUE TITLE
Week of 4/13/15

### ISSUE THREAD

**Issue by hellohynes** [central-pa-cowork/cowork-planning#51]
It's mid-week. Chances are slim. I'm thinking about heading out to Little Amps Friday morning. Nothing's definite though. Company welcome.

**Comment by dustinleer** [comment]
Might be a possibility.

**Comment by jglovier** [comment]
Yeah Friday morning could work for me too, in fact.

**Comment by dustinleer** [comment]
@jglovier @hellohynes tomorrow morning still on?

**Comment by jglovier** [comment]
@dustinleer turns out I have two meetings come up tomorrow that will prevent me from being able to cowork. :frowning:

**Comment by dustinleer** [comment]
@jglovier would you have time to chat tomorrow, the Lancaster WP Meetup group would like to have you come talk about GitHub.

**Comment by hellohynes** [comment]
💔

Actually I have had a meeting come up last minute as well. We are horrible at meeting up with you @dustinleer. @jglovier I feel like we owe Dustin a solid week of co-working dates. I am “currently” free next Tuesday, Wednesday, as well as Friday morning.

I’d like to hear how your interview/talk with Capital One went Dustin.

**Comment by dustinleer** [comment]
@hellohynes it was quite an intense day of interviews, I had 6 interviews if I remember correctly. I think it went well, I won't really know anything until net week, but the people and team I'd work with are pretty cool. The environment of the office has a startup or tech company feel.

As far as days that work for me next week I could do any day, though I may be going to Philly on Friday for Creative Mornings.

**Comment by jglovier** [comment]
> We are horrible at meeting up with you @dustinleer. @jglovier I feel like we owe Dustin a solid week of co-working dates. I am “currently” free next Tuesday, Wednesday, as well as Friday morning.

Yeah, I think I can make next week work. TBH earlier in the week is usually better for me because end of the week I'm usually trying to catch up on all the things I didn't get done yet lol.

> @jglovier would you have time to chat tomorrow

@dustinleer sure - hollar at me whenever.

> @hellohynes it was quite an intense day of interviews, I had 6 interviews if I remember correctly. I think it went well, I won't really know anything until net week, but the people and team I'd work with are pretty cool. The environment of the office has a startup or tech company feel.

@dustinleer awesome man - hope everything works out for you! :tada:
